src/main/java/de/tum/in/www1/exerciseapp/domain/DragAndDropSubmittedAnswer.java

```diff
@@ -70,6 +70,47 @@ public DragItem getSelectedDragItemForDropLocation(DropLocation dropLocation) {
         return null;
     }
 
+    /**
+     * Check if a dragItem or dropLocation were deleted and delete reference to in mappings
+     * @param question the changed question with the changed DragItems and DropLocations
+     */
+    private void checkForDeletedMappings(DragAndDropQuestion question) {
```

Comment:
this method name "checkForDeletedMappings" is confusing, because something is actually removed. Please rename the method name